services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -106,6 +125,16 @@ sealed trait SeqStringWorkflowQueryKey extends WorkflowQueryKey[Seq[String]] {
   }
 }
 
+sealed trait SeqLabelWorkflowQueryKey extends WorkflowQueryKey[Seq[Label]] {
```

Comment:
This code looks strikingly similar to that 10 lines above it.  😄   Seems like this could be DRYed up.